server/space-repository.go

```diff
@@ -125,6 +126,7 @@ func (r *SpaceRepository) GetAllInTime(locationID string, enter, leave time.Time
 			enter, _ := time.Parse(timeFormat, tokens[2])
 			leave, _ := time.Parse(timeFormat, tokens[3])
 			entry := &SpaceAvailabilityBookingEntry{
+				ID:        e.ID,
```

Comment:
This is the space's ID, not the booking ID. Can be fixed by changing line 11 to:

`"ARRAY(SELECT CONCAT(users.id, '@@@', users.email, '@@@', bookings.enter_time, '@@@', bookings.leave_time, '@@@', bookings.id) FROM bookings INNER JOIN users ON users.id = bookings.user_id WHERE "+subQueryWhere+" ORDER BY bookings.enter_time ASC) "+`

Then, you can add the BookingID using;

`BookingID: tokens[4],`